molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerAttributeEditForm.vue

```diff
@@ -1,11 +1,104 @@
 <template>
-  <div>
-    Attribute edit stuff here
+  <div class="row">
+    <div class="col-md-3 attribute-tree">
+      <strong>Attributes</strong>
+      <button @click="addAttribute" class="btn btn-primary btn-sm float-right">Add attribute</button>
+
+      <hr>
+
+      <attribute-tree :selectedAttribute="selectedAttribute" :attributes="attributeTree"
+                      :onAttributeSelect="onAttributeSelect"></attribute-tree>
+      <p v-if="editorEntityType.parent !== undefined">
+        Parent attributes from <strong>{{editorEntityType.parent.label}}:</strong><br>
+        <span v-for="attribute in editorEntityType.parent.attributes">{{attribute.label}}</span>
+      </p>
+    </div>
+
+    <div class="col-md-9 attribute-edit-form" v-if="selectedAttribute !== undefined">
+      <div class="row">
+        <div class="col">
+          <div class="form-group row">
+            <label for="editor-entity-type-label" class="col-2 col-form-label">Attribute</label>
+            <div class="col">
+              {{selectedAttribute.label}}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label for="editor-entity-type-label" class="col-4 col-form-label">Label</label>
+            <div class="col">
+              <input :value="selectedAttribute.label" @input="updateLabel" class="form-control" type="text"
+                     id="editor-entity-type-label">
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label for="editor-entity-type-description" class="col-4 col-form-label">Description</label>
+            <div class="col">
+              <input :value="selectedAttribute.description" @input="updateDescription" class="form-control"
+                     type="text"
+                     id="editor-entity-type-description">
+            </div>
+          </div>
+        </div>
+
+        <div class="col">
+          <!-- TODO nullable, auto, visible, unique, read-only, aggregatable -->
+        </div>
+      </div>
+    </div>
   </div>
 </template>
 
+<style>
+  /*screen-md border on inner column when columns aligned next to each other*/
+  @media (min-width: 768px) {
```

Comment:
I think we should use bootstrap (@include media-breakpoint-up(md) { ... }) instead of recreating the bootstrap functionality.